extension/css/cryptup.css

```diff
@@ -365,6 +365,7 @@ body#settings {
   position: relative;
   font-family: 'Open Sans', sans-serif;
   font-size: 16px;
+  overflow: hidden;
```

Comment:
Removing `overflow` makes part of modal fully hidden on smaller screens:

<img width="700" alt="Screenshot 2024-05-01 at 12 04 20" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/41b94319-1ed1-4670-975f-86035e82c1c9">

Better solution will be to decrease vertical padding for `div#content.dialog.add_pubkey` - currently it's 40px, probably decreasing to 30px will remove unneeded scroll.

Reply:
On this video only horizontal resolution is changed, as there are no displays with 1024x1180 resolution :)
My laptop has 1280x800 resolution, you can try to set it in chrome and check how modal looks there